src/util.hpp

```diff
@@ -496,8 +496,7 @@ inline bool contains(const Container & container, const Value & value)
 
 
 #if 1
-# include <SDL_types.h>
-typedef Sint32 fixed_t;
+typedef int32_t fixed_t;
```

Comment:
Why not just use raw type instead of a typedef? There's plenty of precedence of a raw type in the rest of the code with Uint32.